pkgs/by-name/st/st-snazzy/package.nix

```diff
@@ -42,10 +44,10 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with lib; {
-    homepage = "https://github.com/siduck76/st";
-    description = "Fork of st with many add-ons";
+    homepage = "https://github.com/siduck/st";
+    description = "snazzy terminal (suckless + beautiful)";
     license = licenses.mit;
-    maintainers = with maintainers; [ ];
+    maintainers = with maintainers; [ AndersonTorres ];
     platforms = platforms.linux;
```

Comment:
You are refactoring the whole package. Drop the `with lib;`, its deprecated for a while now.

Reply:
Sidenote: Don't readd AndersonTorresas maintainer without asking! This is a no-go.

Reply:
I don't know why I thought they were the original maintainer.
Thanks for review.